faucet/src/faucet/simple_faucet.rs

```diff
@@ -27,7 +27,7 @@ pub struct SimpleFaucet {
     active_address: SuiAddress,
 }
 
-const DEFAULT_GAS_BUDGET: u64 = 1000;
+const DEFAULT_GAS_BUDGET: u64 = 50000;
```

Comment:
This is for paying the transfer operations instead of sending to users. The latter is defined [here](https://github.com/MystenLabs/sui/blob/9646f9ad79a10fe0e003f5e2c8f020428b182261/faucet/src/main.rs#L44) and configurable in the [deployment script](https://github.com/MystenLabs/sui-operations/blob/ac15d1399d0fec1aef08a052831f6d266b2767c6/docker/validator/entry.sh#L45). You can change this [line](https://github.com/MystenLabs/sui/blob/9646f9ad79a10fe0e003f5e2c8f020428b182261/faucet/src/main.rs#L43) to 10000 and it will take effect after the next deployment(let me know if you want a deployment asap)